src/oaklib/implementations/gilda.py

```diff
@@ -67,24 +66,28 @@ def annotate_text(
         """
         if not configuration:
             raise NotImplementedError("Missing text annotation configuration")
-        if not configuration.matches_whole_text:
-            raise NotImplementedError("Gilda annotator can't be used to match partial text")
+        if configuration.matches_whole_text:
+            yield from self._ground(text)
+        else:
+            yield from self._gilda_annotate(text)
```

Comment:
The GitHub action is failing at this point. This maybe because the `stopwords` resource from `nltk` is not available here. You may want to add a check: if-exists-else-download (`nltk.download('stopwords')`) in `gilda` somewhere before [this step](https://github.com/indralab/gilda/blob/125ab6597a7a65442f74cfcbbdf94fc9e5f102ec/gilda/ner.py#L61C18-L61C33).